client/src/ui/ReviewForm.tsx

```diff
@@ -51,6 +52,16 @@ export default function ReviewForm({
   const [selectedMajors, setMajorSelected] = useState<string[]>(
     value?.major || []
   );
+  const [checked1, setChecked1] = useState();
```

Comment:
I know this is still a WIP, but we should use better variable names here. Great work overall though, this feature is going to be incredibly useful!